packages/express/README.md

```diff
@@ -93,15 +93,15 @@ With this method you can render HTML pages for all configured [`locations`](http
 
 `@untool/express` defines a couple of settings as a preset for `@untool/core`'s [configuration engine](https://github.com/untool/untool/blob/master/packages/core/README.md#configuration). You can manage and access them using the mechanisms outlined there.
 
-| Property    | Type               | Default                         |
-| ----------- | ------------------ | ------------------------------- |
-| `https`     | `boolean`/`Object` | `false`                         |
-| `host`      | `string`           | `process.env.HOST || '0.0.0.0'` |
-| `port`      | `number`           | `process.env.PORT || 8080`      |
-| `locations` | `[string]`         | `[]`                            |
-| `basePath`  | `string`           | `''`                            |
-| `assetPath` | `string`           | `'<basePath>'`                  |
-| `buildDir`  | `string`           | `'<rootDir>/dist'`              |
+| Property    | Type               | Default            |
+| ----------- | ------------------ | ------------------ |
+| `https`     | `boolean`/`Object` | `false`            |
+| `host`      | `string`           | `[HOST]`           |
```

Comment:
Do you see any way we could keep default values with this setup? Or is that something we want to have support for at all?

Reply:
I do not really think we need (configurable) default values. At one point, I had a working implementation of this feature (`<foo|defaultFoo>`, `[FOO|defaultFoo]`), but in the end, I was not convinced this is an essential feature.

Reply:
I have now added support for [dotenv](https://www.npmjs.com/package/dotenv), so I would argue we have a rather nice default value mechanism. Do you agree?